examples/sandbox/index.html

```diff
@@ -8,409 +8,410 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" />
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/milligram/1.3.0/milligram.min.css" />
     <style>
-      .container {
-        margin-top: 2em;
-        padding: 1em;
-        border: #ccc solid 1px;
-        border-radius: 6px;
-      }
+        .container {
```

Comment:
Need to get prettier to format this file. It's currently ignoring for some reason